patches/api/0188-Add-effect-to-block-break-naturally.patch

```diff
@@ -5,15 +5,24 @@ Subject: [PATCH] Add effect to block break naturally
 
 
 diff --git a/src/main/java/org/bukkit/block/Block.java b/src/main/java/org/bukkit/block/Block.java
-index 5ac36e0f90d0889853736390877aa92ec0ca181b..786b8011e98b2fe93cc2418d624f6350ede62d90 100644
+index 5ac36e0f90d0889853736390877aa92ec0ca181b..03ac05c4eb918bf268a68092ad0e95a2c0b89ee4 100644
 --- a/src/main/java/org/bukkit/block/Block.java
 +++ b/src/main/java/org/bukkit/block/Block.java
-@@ -470,6 +470,18 @@ public interface Block extends Metadatable {
+@@ -470,6 +470,27 @@ public interface Block extends Metadatable {
       */
      boolean breakNaturally(@Nullable ItemStack tool);
  
 +    // Paper start
 +    /**
++     * Breaks the block and spawns items as if a player had digged it regardless
++     * of the tool.
++     *
++     * @param triggerEffect Play the block break particle effect and sound
++     * @return true if the block was destroyed
++     */
++    boolean breakNaturally(boolean triggerEffect);
++
++    /**
 +     * Breaks the block and spawns items as if a player had digged it with a
```

Comment:
Pretty minor but is it digged or dug? Maybe even broken? Digged just doesn’t sound right…

Reply:
Should it really be changed? I copied the javadoc from the first breakNaturally method.

Reply:
Question for someone a but higher up i guess, i’m not a fan but it is just the javadoc comment - it makes little difference either way 🤷‍♂️

Reply:
"Breaks the block and spawns item drops as if the player had broken it with the given tool" reads better IMO

Reply:
I have changed it to a small variation of that.